src/assets/music/play-assets.js

```diff
@@ -67,7 +67,7 @@ const fun = {
     },
     11: {
         name: 'Danger zone',
-        link: 'https://www.youtube.com/watch?v=58QOBqAWNzE',
+        link: 'https://www.youtube.com/watch?v=siwpn14IE7E',
         volume: 0.04
```

Comment:
None of the volumes have been adjusted for any of these links, someone will either need to QA this and change the volume's for the links as required or we can complete the volume normalisation that will hopefully be implemented with #18